common/src/event.rs

```diff
@@ -31,27 +31,27 @@ use scale_info::TypeInfo;
 /// Same as `gear_core::message::DispatchKind`,
 /// but with additional info about reply.
 #[derive(Debug, Encode, Decode, Clone, PartialEq, Eq, TypeInfo)]
-pub enum Entry {
+pub enum MessageEntry {
     /// Init entry point.
-    Init,
+    InitEntry,
     /// Handle entry point.
-    Handle,
+    HandleEntry,
     /// Handle reply entry point.
-    Reply(MessageId),
+    HandleReplyEntry(MessageId),
     /// System signal entry point.
-    Signal,
+    HandleSignalEntry,
 }
 
 /// Status of dispatch dequeue and execution.
 #[derive(Debug, Encode, Decode, Clone, PartialEq, Eq, TypeInfo)]
 pub enum DispatchStatus {
     /// Dispatch was dequeued and succeed with execution.
-    Success,
+    DispatchExecuted,
     /// Dispatch was dequeued and failed its execution.
-    Failed,
+    DispatchExecutionFailed,
     /// Dispatch was dequeued and wasn't executed.
     /// Occurs if actor no longer exists.
-    NotExecuted,
+    DispatchNotExecuted,
```

Comment:
The same thing, prefix `Dispatch` isn't needed I guess. And actually Success and Failed look more understandable